packages/client-core/src/world/reducers/scenes/selector.ts

```diff
@@ -1,7 +1,7 @@
-import { createSelector } from 'reselect'
```

Comment:
Please delete this file instead of commenting out if it's not going to be used now